### DISCUSSION TITLE
Question: The Potential of Polymodding

### DISCUSSION THREAD

**Discussion by Johferson** [FunkinCrew/Funkin#4046] [Q&A] [answered]
[weed]: <> (This isn't a place for AMA type questions, if you want to ask any of the devs something, reach out to them on twitter prob )
[weed]: <> (any biz bullshit can go to cameron.taylor.ninja@gmail.com)

## What is your question?
Hi! Since it has been a week since 0.3.x was released and introduced Haxe's polymod support (along with the extreme ease of building stages and characters), I couldn't really help but ask...

How powerful is FNF's HScript system? Can it overhaul UI stuff? Do in-game rigging for custom characters (just like in the base game)? Basically everything?

I made this question since people are still modding this game via forked source code (Psych) or if not, just doing ports of already existing mods. Thank you!

**Comment by biomseed** [comment, marked as answer]
You can actually do all of those things and more, it's just that people are still getting used to it and aren't used to things like hscript and polymod so they don't know the potential of it
Reason people still use psych/hardcoding is because they don't actually know that this can actually do a lot, same with the modders who are porting mods currently, but when people start making big mods with code and stuff, it's probably going to pick up fast
There's also people who are still used to psych engine and stuff and think the new update's modding is buggy (and it kind of is at the moment)

**Comment by ninjamuffin99** [comment]
i believe biomseed answered it well 🙂 polymod is very nice, weekend 1 gameplay is all polymod script ! also gonna close this issue !
